src/Core/Components/List/FluentCombobox.razor.cs

```diff
@@ -41,6 +41,13 @@ public partial class FluentCombobox<TOption> : ListComponentBase<TOption> where
         .AddStyle("min-width", Width, when: !string.IsNullOrEmpty(Width))
         .Build();
 
+    public override async Task SetParametersAsync(ParameterView parameters)
```

Comment:
I think you could use this code to avoid to use "async/await" in this case.

```csharp
public override Task SetParametersAsync(ParameterView parameters)
{
     parameters.SetParameterProperties(this);
     return base.SetParametersAsync(ParameterView.Empty);
}
```